application/src/main/java/org/togetherjava/tjbot/features/mathcommands/wolframalpha/WolframAlphaImages.java

```diff
@@ -66,10 +65,10 @@ static BufferedImage renderSubPod(SubPod subPod) {
         Graphics graphics = destinationImage.getGraphics();
 
         try {
-            graphics.drawImage(ImageIO.read(new URL(sourceImage.getSource())), IMAGE_MARGIN_PX,
-                    IMAGE_MARGIN_PX, null);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
+            graphics.drawImage(ImageIO.read(new URI(sourceImage.getSource()).toURL()),
+                    IMAGE_MARGIN_PX, IMAGE_MARGIN_PX, null);
+        } catch (IOException | URISyntaxException e) {
+            throw new RuntimeException(e);
         }
```

Comment:
ig u'll have to catch and throw both separately

Reply:
why are these code changes in this PR? are they required for java 20?